AnkiConnect.py

```diff
@@ -417,6 +417,22 @@ def deckNames(self):
             return collection.decks.allNames()
 
 
+    def launchBrowser(self, query):
+        browser = aqt.dialogs.open("Browser", self.window())
+        browser.activateWindow()
+        if query:
+            query = u'"{}"'.format(query)
```

Comment:
The Anki alpha uses Python 3 which does not define `u` prefix for unicode strings (all strings are unicode by default), so this code will break. Use the `unicode` wrapper defined in the file instead:
```
query = unicode('"{}"'.format(query))
```